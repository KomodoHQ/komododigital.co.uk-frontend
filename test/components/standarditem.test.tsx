import React from 'react';
import StandardItem from '../../src/components/standarditem';
import renderer from 'react-test-renderer';

test('StandardItem element is rendered', () => {
  const data = {
  };
  const component = renderer.create(<StandardItem imgsrc={{childImageSharp:{fixed: {width: 1, srcSet:'',src:'',height:1}}}} link="/About" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
