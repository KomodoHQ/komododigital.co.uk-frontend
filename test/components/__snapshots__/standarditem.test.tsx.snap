// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StandardItem element is rendered 1`] = `
<div
  className="StandardItem"
>
  <a
    href="/About"
  >
    <div
      className=" gatsby-image-wrapper"
      style={
        Object {
          "display": "inline-block",
          "height": 1,
          "overflow": "hidden",
          "position": "relative",
          "width": 1,
        }
      }
    >
      <picture>
        <source
          srcSet=""
        />
        <img
          alt=""
          height={1}
          onLoad={[Function]}
          src=""
          style={
            Object {
              "height": "100%",
              "left": 0,
              "objectFit": "cover",
              "objectPosition": "center",
              "opacity": 1,
              "position": "absolute",
              "top": 0,
              "transition": "opacity 0.5s",
              "width": "100%",
            }
          }
          width={1}
        />
      </picture>
      <noscript
        dangerouslySetInnerHTML={
          Object {
            "__html": "<picture><img width=\\"1\\" height=\\"1\\" src=\\"\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
          }
        }
      />
    </div>
  </a>
</div>
`;
